packages/app/components/Nav/AlertsContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { API } from "@koh/api-client";
+import { AlertType, RephraseQuestionPayload } from "@koh/common";
+import React, { ReactElement } from "react";
+import useSWR from "swr";
+import { useRouter } from "next/router";
+import StudentQuestionRephraseModal from "../Queue/Student/StudentQuestionRephraseModal";
+
+type AlertsContainerProps = {
+  courseId: number; // TODO: replace this with a courseId that comes from the RephraseQuestionPayload
```

Comment:
Course should match the course that the alert originated from, this courseId is coming from the current course in the navbar (not what we want)